zerver/lib/narrow.py

```diff
@@ -336,7 +342,13 @@ def by_is(self, query: Select, operand: str, maybe_negate: ConditionTransform) -
         assert not self.is_web_public_query
         assert self.user_profile is not None
 
+        if operand == "dm":
+            cond = column("flags", Integer).op("&")(UserMessage.flags.is_private.mask) != 0
+            return query.where(maybe_negate(cond))
         if operand == "private":
+            # "private" is a deprecated operand with the "is" operator.
+            # For backwards-compatibility with links/URLs, we currently
+            # maintain support for fetching messages with these narrows.
             cond = column("flags", Integer).op("&")(UserMessage.flags.is_private.mask) != 0
             return query.where(maybe_negate(cond))
```

Comment:
Would it make sense to write this as `operand in ["dm", "private"]`? I mainly ask because it's harder to mess up updating that code by forgetting that there's names for this operator.

(The query applies to both hunks in this file).

Reply:
Yeah, that's much easier to read and maintain. Updated as such.